def test_assert_called_with_message(self):
    mock = Mock()
    self.assertRaisesRegexp(AssertionError, 'Not called', mock.assert_called_with)
